plugins/rias/ls.rias.js

```diff
@@ -83,6 +84,7 @@
 
 		setOption = function(attr, run){
 			var attrVal = elem.getAttribute('data-'+ attr);
+			attrVal = elemStyles.getPropertyValue('--' + attr).trim() || attrVal;
```

Comment:
I would prefer a prefix (maybe `--ls-`) here (custom variables do inherit (if you don't stop it explicitly) and this could produce some conflicts.